workspaces/vscode-python/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# noble is the codename for Ubuntu 24.04 LTS
+FROM codercom/code-server:4.90.3-noble
+# Skip code-server:4.91.0-noble and delay updating beyond this until
+# code-server catches up to vscode 1.91.1 because of possible issues on
+# macOS. See: https://github.com/microsoft/vscode/issues/219893
```

Comment:
If this issue is unique to macOS, how could it possibly affect PL where the Docker image will always be running on Linux (either natively on Linux machines, or in the Linux VM locally on macOS/Windows)?

Reply:
I'd thought the issue was specific to _Safari_ but I may have misunderstood that when I viewed the issue discussions.

Reply:
The promised version with the fix has already propagated to code-server to I'll try just updating the base image now, and then we can remove the warning.